airbyte-integrations/connectors/source-postgres/src/test-integration/java/io/airbyte/integrations/io/airbyte/integration_tests/sources/PostresSourceDatatypeTest.java

```diff
@@ -294,17 +293,13 @@ protected void initTests() {
                 "08:00:2b:01:02:03:04:07")
             .build());
 
-    // The Money type fails when amount is > 1,000. in JdbcUtils-> rowToJson as r.getObject(i);
-    // Bad value for type double : 1,000.01
-    // The reason is that in jdbc implementation money type is tried to get as Double (jdbc
-    // implementation)
     // Max values for Money type: "-92233720368547758.08", "92233720368547758.07"
     addDataTypeTestData(
         TestDataHolder.builder()
             .sourceType("money")
-            .airbyteType(JsonSchemaPrimitive.STRING)
-            .addInsertValues("null", "'999.99'")
-            .addExpectedValues(null, "999.99")
+            .airbyteType(JsonSchemaPrimitive.NUMBER)
+            .addInsertValues("null", "'999.99'", "'1001.01'", "'-92233720368547758.08'", "'92233720368547758.07'")
```

Comment:
how can we make sure this parses does it make sense to also add something with some currency symbols e.g: `$1,000`  to ensure it is working correctly?

Reply:
Done.

Reply:
Well, the values are inserted into an Postgres instance (via test container). It is Postgres that should guarantee the correctness of such insertion operations. So actually we should not need to test inserting a money value with the currency symbol.